apps/editorial_features/components/eoy/templates/carousel.jade

```diff
@@ -0,0 +1,16 @@
+.carousel.mgr-horizontal-nav
+  .mgr-cells.js-mgr-cells
+    .mgr-cell.js-mgr-cell
+      img(src='https://artsy-vanity-files-production.s3.amazonaws.com/images/float_01.jpg' width='800')
+    .mgr-cell.js-mgr-cell
+      img(src='https://artsy-vanity-files-production.s3.amazonaws.com/images/float_02.jpg' width='800')
+    .mgr-cell.js-mgr-cell
+      img(src='https://artsy-vanity-files-production.s3.amazonaws.com/images/float_03.jpg' width='800')
+    .mgr-cell.js-mgr-cell
+      img(src='https://artsy-vanity-files-production.s3.amazonaws.com/images/float_04.jpg' width='800')
+    .mgr-cell.js-mgr-cell
+      img(src='https://artsy-vanity-files-production.s3.amazonaws.com/images/float_05.jpg' width='800')
+    .mgr-cell.js-mgr-cell
+      img(src='https://artsy-vanity-files-production.s3.amazonaws.com/images/float_06.jpg' width='800')
+  nav.mgr-navigation.js-mgr-navigation
+    include ../../../../../components/merry_go_round/templates/horizontal_navigation
```

Comment:
Thanks for separating this. Let's eventually get these image configurable in Writer.